### DISCUSSION TITLE
Can't start nightly build

### DISCUSSION THREAD

**Discussion by nine-testabe** [keycloak/keycloak#15104] [Q&A]
Hi all
I have downloaded the nightly build keycloak-999-SNAPSHOT.zip
from https://github.com/keycloak/keycloak/releases/tag/nightly
But when I try to start the server with
.\bin\kc.bat start-dev --log-level=INFO
I get:
Error: Could not find or load main class io.quarkus.bootstrap.runner.QuarkusEntryPoint

I understand it's the bleeding edge, but does someone know how to make it start ?

**Comment by stianst** [comment]
Just tried that here and it's working just fine. Was the download corrupted or something?

**Reply by nine-testabe** [reply]
No, the zip file decompressed just fine. Just tried to download again - same result.
The java JVM can't find QuarkusEntryPoint in the classpath.
I can see in the bat file, that the bat file depend on external java. Which version of java does it expect ?

**Reply by avillc-tech-research** [reply]
I am also facing the same issue with version 20.0.0 of keycloak, do we need to install anything else other than jdk 11 or higher?
